src/copy.c

```diff
@@ -361,6 +361,12 @@ timescaledb_CopyFrom(CopyState cstate, Relation main_rel, List *range_table, Hyp
 	ExecResetTupleTable(estate->es_tupleTable, false);
 
 	ExecCloseIndices(resultRelInfo);
+
+#if PG10
```

Comment:
So I saw this on 9.6 as well. Should this be unguarded? or an alternative for 9.6 given?

Reply:
Yeah, we should definitely check whether this is a general thing.

Reply:
Fixed. Look again?